### ISSUE TITLE
Standardize app sizes

### ISSUE THREAD

**Issue by belgamo** [wendelfreitas/animavita#209]
### Task
Currently, our UI is looking different across devices with different density pixels.

### Suggested approach
We could adapt our theme config to define sizes.

https://medium.com/soluto-engineering/size-matters-5aeeb462900a

**Comment by PabloSSena** [comment]
Hello @belgamo can i work on this?

**Comment by belgamo** [comment]
Hello @PabloSSena, any updates here? Can I assign it to someone else?

**Comment by Adrian-kx** [comment]
hey @belgamo  can you please assign this task to me?

**Comment by belgamo** [comment]
> hey [@belgamo](https://github.com/belgamo) can you please assign this task to me?

done

**Comment by Adrian-kx** [comment]
Thank you, I would just like to confirm whether the approach below makes sense and is aligned with what you recommend:
	•	Use flexbox and percentages for everything related to layout (structure, widths, positioning, spacing between main blocks).
	•	Use scaling functions only for values that are not layout-related, such as fontSize, lineHeight, borderRadius, icons, paddings, and small spacing details.

For scaling, I plan to use utilities like these:

```ts
import { Dimensions } from 'react-native';
const { width, height } = Dimensions.get('window');

const guidelineBaseWidth = 350;
const guidelineBaseHeight = 680;

const scale = size => width / guidelineBaseWidth * size;
const verticalScale = size => height / guidelineBaseHeight * size;
const moderateScale = (size, factor = 0.5) =>
  size + (scale(size) - size) * factor;

export { scale, verticalScale, moderateScale };
```

In summary:
	•	Percentages / flex for layout
	•	moderateScale (or similar) for typography and fine adjustments

Do you consider this approach to be correct and a good practice for the application?

**Comment by belgamo** [comment]
> Thank you, I would just like to confirm whether the approach below makes sense and is aligned with what you recommend: • Use flexbox and percentages for everything related to layout (structure, widths, positioning, spacing between main blocks). • Use scaling functions only for values that are not layout-related, such as fontSize, lineHeight, borderRadius, icons, paddings, and small spacing details.
> 
> For scaling, I plan to use utilities like these:
> 
> import { Dimensions } from 'react-native';
> const { width, height } = Dimensions.get('window');
> 
> const guidelineBaseWidth = 350;
> const guidelineBaseHeight = 680;
> 
> const scale = size => width / guidelineBaseWidth * size;
> const verticalScale = size => height / guidelineBaseHeight * size;
> const moderateScale = (size, factor = 0.5) =>
>   size + (scale(size) - size) * factor;
> 
> export { scale, verticalScale, moderateScale };
> In summary: • Percentages / flex for layout • moderateScale (or similar) for typography and fine adjustments
> 
> Do you consider this approach to be correct and a good practice for the application?

LGTM! i'm open to suggestions though

**Comment by Adrian-kx** [comment]
Perfect, I’ll proceed with this approach then.
